app/views/shared/_landing_strip.html.haml

```diff
@@ -1,6 +1,6 @@
 .landing-strip
   = t('landing_strip_html',
     name: content_tag(:span, display_name(account), class: :emojify),
-    domain: site_hostname,
-    link_to_root_path: link_to(content_tag(:strong, site_hostname), root_path),
-    sign_up_path: new_user_registration_path)
+    link_to_root_path: link_to(content_tag(:strong, site_hostname), root_path))
+  - if open_registrations?
```

Comment:
It would be better to use [`InstancePresenter`](https://github.com/tootsuite/mastodon/blob/a588358f41f36b2df1dcc4bb2c3df6c54ed52f88/app/presenters/instance_presenter.rb) like [`about/_links.html.haml`](https://github.com/tootsuite/mastodon/blob/a588358f41f36b2df1dcc4bb2c3df6c54ed52f88/app/views/about/_links.html.haml).